jira_agile_metrics/calculators/defects.py

```diff
@@ -77,120 +90,198 @@ def run(self):
 
     def write(self):
         chart_data = self.get_result()
+        # fixversion_field_choice = self.settings['defects_fixversion_label']
+        # print(fixversion_field_choice)
```

Comment:
Please don't leave dead code commented out - remove if not needed

Reply:
I will fix that.